about.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/e32d72064d.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>The Complete 2022 Web Development Bootcamp</title>
+    <link rel="icon" type="image/x-icon" href="images/favicon.ico?v=2" />
+  </head>
+  <body>
+    <!-- MOBILE-POPUP -->
+    <div class="mobile-menu-background">
+      <i class="fa-solid fa-x close-menu"></i>
+      <ul>
+        <li><a class="close-menu-item1" href="index.html#home">Home</a></li>
+        <li><a class="close-menu-item2" href="#header">About</a></li>
+        <li><a class="close-menu-item3" href="#about-featured">Featured</a></li>
+      </ul>
+    </div>
+
+    <!-- NAVIGATION BAR -->
+    <section class="nav-top">
+      <div>
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <span>English</span>
+        <span>My page</span>
+        <span>Logout</span>
+      </div>
```

Comment:
- [ ] (Optional) This is not a hard requirement, you can change the `div` tag to a `nav` tag for HTML semantic purposes. It is still a nav bar although it is the upper level. 👍